test/cppsuite/test_harness/core/component.h

```diff
@@ -56,14 +60,36 @@ class component {
     virtual void
     load()
     {
+        debug_print("Loading component: " + _name, DEBUG_INFO);
         _enabled = _config->get_optional_bool(ENABLED, true);
+        _throttle = throttle(_config->get_optional_int(OPS_PER_SECOND, 1));
```

Comment:
Instead of the hardcoded value, is there a way to retrieve it from the config definition ?

Reply:
I'll investigate this one this morning, we should be able to.